dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/utils/MiscUtils.kt

```diff
@@ -13,6 +13,14 @@ import com.google.gson.JsonObject
 import com.google.gson.JsonPrimitive
 import java.util.Date
 
+internal fun retryWithDelay(
```

Comment:
why not just changing the original function signature ?

Reply:
Because I don't want to touch at the legacy implementation yet.